docs/user/quickstart.rst

```diff
@@ -52,6 +52,12 @@ it on a toy classification dataset using skorch
     net.fit(X, y)
     y_proba = net.predict_proba(X)
 
+.. note::
+
+    The use of ``softmax`` might result in numerical instability. If you have
```

Comment:
In general, I think using `CrossEntropyLoss` is the recommended approach. What do you think about updating the example to use `CrossEntropyLoss` and removing the `softmax`?

Reply:
As always, you come up with the best suggestions. I changed the example accordingly.